### ISSUE TITLE
elastic search as the primary reporting db? 

### ISSUE THREAD

**Issue by mallorybobalice** [spender-sandbox/cuckoo-modified#340]
Sooo, I recall a few months ago the opinion was ES as primary wasn't stable. 
Has this changed with the recent distributed cuckoo commits? 

I'd like to switch but 2 things kinda worry me. 

1)I use Mongo plus search only ES. And see a few insert errors a day. I didn't look at it to closely, but hopefully that means attribute insert issues not whole task reports. Is that the same for es as primary db? 
2)in search only none of the fields are tagged as timestamps... so can't do nice time series related features... why aren't things like submission, or analysis or processing start end.? 
Would also be good to add analysis time and processing time to easily dashboard them

**Comment by mallorybobalice** [comment]
Ps anyone try es 5? Seems to work ok with search only, upgraded the test box this week

**Comment by doomedraven** [comment]
No, distributed cuckoo not changes nothing, we support insert, but we dont care about scheme mappint as in core is also not cared
